trillian/integration/ct_lifecycle_test.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Should this just be part of the ct multi server integration test?
Can provision a new tree into the already running instances - otherwise you'll adding another minute or so to the travis time while it waits for the old jobs to stop, one-by-one, then starts new identical jobs, waits, runs the test, waits for all the jobs to die again, etc...

Reply:
The scripts currently do one setup / provision step during the setup and run the test against all the logs in the config so it was easier to do it this way. That could be changed I suppose.

Reply:
It seems like an "easy win" to keep Travis times under control - if you prefer to do it separately that's cool too, though - just pop a TODO in there.

Reply:
One issue here is that the test sets up 3 signers. In a normal integration test there aren't enough submissions for them to fight over work but with this test there is. On the plus side it doesn't break anything when there's contention, just creates a lot of extra log noise.

Reply:
I'm going to look at this next but I think it could be quite a lot of work.

Reply:
I managed to come up with a (fairly) painless way of doing this. It's in #218.